pkg/db/db.go

```diff
@@ -46,11 +51,19 @@ func FromEnv(path string, backend string) (datab, error) {
 	if path == "" {
 		return nil, fmt.Errorf("missing DB_PATH")
 	}
-
-	c, err := newDB(config{
+	var (
+		c   datab
+		err error
+	)
```

Comment:
A cleaner way to do this is you can name the return variables in the function signature.

```
func FromEnv(path string, backend string, useCloudSQL bool) (c datab, err error) {
```

It essentially does what you're doing but at the start of the function